opm/autodiff/SimulatorFullyImplicitBlackoilOutput.cpp

```diff
@@ -296,8 +296,21 @@ namespace Opm
             }
             else {
                 // just write the data to disk
-                writeTimeStepSerial( timer, state, wellState, cellData, substep );
+                int err = 0;
+                try {
+                    writeTimeStepSerial( timer, state, wellState, cellData, substep );
+                    MPI_Bcast(&err, 1, MPI_INT, 0, MPI_COMM_WORLD);
+                } catch (std::runtime_error& msg) {
```

Comment:
i did test it and it did work but i agree that's at the mercy of broadcast internals, and thus a rather naughty hack. it's paired with another bcast call below that the other procs enters. it is only a draft, i requested some instructions where to dig out comm and such but i got no response.